lib/aliquot/validator.rb

```diff
@@ -119,15 +119,15 @@ def self.messages
       optional(:cryptogram).filled(:str?)
       optional(:eciIndicator).filled(:str?, :eci?)
 
-      rule('when authMethod is CRYPTOGRAM_3DS, cryptogram': %i[authMethod cryptogram]) do |method, cryptogram|
-        method.eql?('CRYPTOGRAM_3DS') > cryptogram.filled?
+      rule(cryptogram: %i[authMethod cryptogram]) do |method, cryptogram|
+        method.eql?('CRYPTOGRAM_3DS') > required(:cryptogram)
       end
 
-      rule('when authMethod is PAN_ONLY, eciIndicator': %i[authMethod eciIndicator]) do |method, eci|
+      rule(eciIndicator: %i[authMethod eciIndicator]) do |method, eci|
         method.eql?('PAN_ONLY').then(eci.none?)
       end
 
-      rule('when authMethod is PAN_ONLY, cryptogram': %i[authMethod cryptogram]) do |method, cryptogram|
+      rule('cryptogram': %i[authMethod cryptogram]) do |method, cryptogram|
```

Comment:
`'cryptogram'` vs `cryptogram`

If you compare this to line 122.

Reply:
On a second note, I can't merge line 122 and 130 into one line, fails for obscure reasons.